linuxdoc/kernel_doc.py

```diff
@@ -1103,6 +1103,25 @@ def output_function_decl(
                 , sec_level = 3
                 , ID = function + "." + header)
 
+        # if no "See Also" section was filled, try to use the :function:
+        # argument list from the kernel-doc directive.
+
+        if "See also" not in sections.keys():
+            content_list = list()
+            if 'Description' in sections:
+                fnlist = re.findall(r'([a-z0-9_]*)\(\)', sections['Description'], re.DOTALL)
+                while function in fnlist:
+                    fnlist.remove(function)  # remove self from the list
+                content_list = [':c:func:`{}`'.format(fn) for fn in fnlist]
+            if content_list:
+                content = ", ".join(content_list)
+                self.write_section(
+                    "See also"
+                    , content
+                    , sec_level = 3
+                    , ID = "fun." + function + "." + "seealso")
+
+
```

Comment:
Sorry, ATM I do not understand for what "See also" is good .. please give me a small kernel-doc example / thanks!

Reply:
This is intended for the man pages (present for all formats, but especially for the man pages).
The idea is to parse the function's description for reference to other functions and use that set to create reference between the functions.

This is useful for example for functions like `xxx_open()` which says in its doc "to be closed by `xxx_close()`". That way, `xxx_open()` references `xxx_close()`

I'll add documentation to kernel-doc, and  all-in-a-tumble.* examples (for this one and the other you requested).